cube/char_set.cpp

```diff
@@ -1,5 +1,5 @@
 /**********************************************************************
- * File:        char_samp_enum.cpp
```

Comment:
Such comments might have been useful 20 years ago, but nowadays we have configuration management and can track all history in Git, so I think there is no need for such comments in the code any more. I'd prefer removing the existing comments (in a later PR of course) instead of adding more of them to reach uniformity.